seed/challenges/01-responsive-web-design/css-flexbox.json

```diff
@@ -1085,39 +1091,38 @@
         "  #box-container {",
         "    background: gray;",
         "    display: flex;",
-        "    ",
-        "    height: 500px;",
+        "    height: 100%;",
         "  }",
```

Comment:
While we're in here, add a blank row right after the `height` property with the same indenting (since all the other challenges put an empty row in the seed code for where the properties should go).